### PR TITLE
Find friends

### PR DESCRIPTION
**Pull request by sawyer1776**
Working on animation, basic layout is done

### CODE REVIEW

File: src/pages/find-friends/index.tsx
Comment (burhan-syed) [inline src/pages/find-friends/index.tsx:1]: Can you remove this file

Comment (sawyer1776) [reply src/pages/find-friends/index.tsx:1]: sure

**Review by burhan-syed — CHANGES_REQUESTED** [review]
Build is failing due to undefined types. Easiest way to resolve this is to move the handlers inline into the JSX or get the type of event from the callback function.

For inputs onChange e is `ChangeEvent<HTMLInputElement>`
For errors the type is probably unknown

**Comment by sawyer1776** [comment]
fixed the type errors

**Review by burhan-syed — APPROVED** [review]

**Pull request merged**